### PR TITLE
Unattach request and response logs for test cases passed

### PR DESCRIPTION
**Pull request by mynguyenlgg**
- Unattach request and response logs for test cases passed
- Use SoftAssert
- Update readme

### CODE REVIEW

File: api-test/src/main/java/com/client/listeners/AllureTestListener.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import io.qameta.allure.model.StepResult;
+import io.qameta.allure.model.TestResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AllureTestListener implements TestLifecycleListener {
+
+    private static final Logger logger = LoggerFactory.getLogger(AllureTestListener.class);
```
Comment (dinhbao10t4) [inline api-test/src/main/java/com/client/listeners/AllureTestListener.java:15]: use @Slf4j

File: api-test/src/test/java/com/client/tests/board/CreateBoardTests.java [hunk 2]
```diff
@@ -17,6 +16,8 @@ public class CreateBoardTests {
     private String boardID;
     private final BoardService boardService = new BoardService();
 
+    SoftHamcrestAssert softHamcrestAssert = new SoftHamcrestAssert();
```
Comment (dinhbao10t4) [inline api-test/src/test/java/com/client/tests/board/CreateBoardTests.java:19]: private SoftHamcrestAssert softHamcrestAssert

File: api-test/src/main/java/com/client/listeners/AllureTestListener.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            try {
+                List<StepResult> steps = result.getSteps();
+                for (StepResult stepResult: steps) {
+                    List<Attachment> attachmentList = stepResult.getAttachments().stream()
+                            .filter(att -> (!att.getName().equals("Request")) && (!att.getName().contains("Response")))
+                            .collect(Collectors.toList());
+                    stepResult.setAttachments(attachmentList);
+                }
+            } catch (Exception e) {
+                logger.error( e.getMessage());
```
Comment (dinhbao10t4) [inline api-test/src/main/java/com/client/listeners/AllureTestListener.java:30]: format code

File: api-test/src/main/java/com/client/listeners/AllureTestListener.java (hunk 1 above)
Comment (mynguyenlgg) [reply api-test/src/main/java/com/client/listeners/AllureTestListener.java]: I updated

File: api-test/src/test/java/com/client/tests/board/CreateBoardTests.java (hunk 2 above)
Comment (mynguyenlgg) [reply api-test/src/test/java/com/client/tests/board/CreateBoardTests.java]: I updated

File: api-test/src/main/java/com/client/listeners/AllureTestListener.java (hunk 3 above)
Comment (mynguyenlgg) [reply api-test/src/main/java/com/client/listeners/AllureTestListener.java]: I updated

**Review by dinhbao10t4 — APPROVED** [review]

**Pull request merged**